stock_manager/scripts/api/sql_api.py

```diff
@@ -0,0 +1,93 @@
+import mysql.connector
```

Comment:
General design question: I think it doesn't matter so much which data source we use, but I'm curious why we're using two. Why are we using both google sheets and also mysql as data sources?